src/shared/components/palette/fixed-list/write-fixed-list-field.component.ts

```diff
@@ -11,19 +11,12 @@ export class WriteFixedListFieldComponent extends AbstractFieldWriteComponent im
   fixedListFormControl: FormControl;
 
   ngOnInit() {
-
-    /**
-     *
-     * Reassigning list_items from formatted_list when list_items is empty for DynamicList's
-     */
-    if (!this.caseField.list_items && this.caseField.formatted_value && this.caseField.formatted_value.list_items) {
-      this.caseField.list_items = this.caseField.formatted_value.list_items;
-    }
-
     let isNull = this.caseField.value === undefined || this.caseField.value === '';
```

Comment:
Could use dual NOT operators to reduce this code, i.e. if (!!this.caseField.value)

Reply:
done